spec/acceptance/site_layout_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'landing page', type: :feature do
+  before do
+    visit root_path
+  end
+  context 'When user not logged in' do
+    it { expect(page).to have_content 'Sign up' }
+    it { expect(page).to have_content 'login' }
+    it { expect(page).to have_css '.image' }
+  end
+
+  context 'When user is logged in' do
+    it { expect(page).to have_content 'Receipts' }
+    it { expect(page).to have_content 'New receipt' }
+  end
+end
```

Comment:
At this point 984e2e2c0071f5948624154e28e02b7cc63209d1, this spec will not pass, right? Try to make it habit to create a commit only when tests are green. So if someone needs to run the application at that point, she knows that she can branch out with a clean tet suite.